armsrc/mifarecmd.c

```diff
@@ -1024,18 +1033,21 @@ void MifareChkKeys(uint16_t arg0, uint32_t arg1, uint8_t arg2, uint8_t *datain)
 		TKeyIndex keyIndex = {{0}};
 		uint8_t sectorCnt = blockNo;
 		res = MifareMultisectorChk(datain, keyCount, sectorCnt, keyType, &auth_timeout, OLD_MF_DBGLEVEL, &keyIndex);
-		if (res >= 0) {

```

Comment:
Don't we need to test `res` and set `reject_next` here as well?

Reply:
As the client side currently does not try to queue keys for multi-sector key checks it's not really needed, but I went ahead and added it anyway.

Reply:
Again you are correct.